### DISCUSSION TITLE
problem in installing

### DISCUSSION THREAD

**Discussion by princedeo** [chipsec/chipsec#2040] [General]
C:\chipsec-1.12.6\chipsec_tools\compression\Include\7zTypes.h(11): fatal error C1083: Cannot open include file: 'stddef.h': No such file or directory
error: command 'C:\\Program Files (x86)\\Microsoft Visual Studio\\2022\\BuildTools\\VC\\Tools\\MSVC\\14.38.33130\\bin\\HostX86\\x64\\cl.exe' failed with exit code 2

**Comment by npmitche** [comment]
Hi @princedeo can you share what command you are using when you see this error? Also any additional logs could prove helpful if you have any.
